src/commonComponents/header/Header.tsx

```diff
@@ -34,7 +35,12 @@ function Header() {
       <header className="header">
         <div className="header__logo-box">
           <NavLink to="/">
-            <img src={logo} className="header__logo" alt="logo" />
+            <img
+              src={logo}
+              className="header__logo"
+              alt="logo"
+              onClick={() => handleNavClick('logo')}
```

Comment:
`                onClick={() => handleNavClick('home')}
`
this will do the job